exporter/azure_monitor/azure_monitor.go

```diff
@@ -3,30 +3,31 @@ package azure_monitor
 // This includes examples on how to create azure exporters to send spans.
 
 import (
-	"fmt"
 	"errors"
+	"time"
+	"fmt" // for debugging
 	"go.opencensus.io/exporter/azure_monitor/common"
 	"go.opencensus.io/trace"
 )
 
 type AzureTraceExporter struct {
-	projectID          string
+	ProjectID          string

```

Comment:
I still don't understand why we need a `ProjectID` here, would you point me to the code where you see this? Thanks.

Reply:
Huh, I can't seem to find it. I thought it was in the python repo, but I guess I saw it somewhere else then. I will go ahead and delete that attribute then.